variables.tf

```diff
@@ -52,6 +52,28 @@ variable "appautoscaling_settings" {
   type        = map(any)
 }
 
+variable "capacity_provider_strategy" {
+  default     = null
+  description = "List of capacity provider strategies."
+  type = list(object({
+    capacity_provider = string
+    weight            = optional(string, null)
```

Comment:
weight is required according to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service#weight